hack/jenkins/job-configs/global.yaml

```diff
@@ -13,6 +13,17 @@
     publishers:
         - postbuildscript:
             builders:
+                # Dump any master/node logs that are still around at this point
```

Comment:
in the normal case, the cluster will have already been torn down by now, so this would usually be a no-op. was that intentional?